src/status/mod.rs

```diff
@@ -1,5 +0,0 @@
-use bindings::{Struct_ompi_status_public_t, MPI_Status};
-
-pub fn new() -> MPI_Status {
```

Comment:
might add this back in the future, but the new() was never really needed, it exists in bindings mod.
mainly removed because default(), not sure why i even added that in the first place.